pyzeebe/task/task_builder.py

```diff
@@ -35,7 +35,8 @@ async def job_handler(job: Job) -> Job:
         original_return_value, succeeded = await run_original_task_function(prepared_task_function, task_config, job)
         job.variables.update(original_return_value)
         job.variables.pop(task_config.job_parameter_name, None)
-        job = await after_decorator_runner(job)
+        await job.set_running_after_decorators_status()
+        await after_decorator_runner(job)
```

Comment:
Is there a reason to not assign the job to what after_decorator_runner returns?

Reply:
No reason, my mistake. I have pushed a fix